docs/collecting-data/collecting-from-own-applications/mobile-trackers/tracking-events/ecommerce-tracking/index.md

```diff
@@ -0,0 +1,702 @@
+`ProductListClick` | Track the click/selection of a product from a product list.
+`PromotionView`    | Track an impression for an internal promotion banner or slider or any other type of content that showcases internal products/categories.
+`PromotionClick`   | Track the click/selection of an internal promotion.
+`CheckoutStep`     | Track a checkout step completion in the checkout process together with common step attributes for user choices throughout the checkout funnel.
+`Transaction`      | Track a transaction/purchase completion.
+`Refund`           | Track a transaction partial or complete refund.
+
+
+Each ecommerce event is a [self-describing](docs/collecting-data/collecting-from-own-applications/mobile-trackers/custom-tracking-using-schemas/index.md) event using a single schema, 
+`iglu:com.snowplowanalytics.snowplow.ecommerce/snowplow_ecommerce_action/jsonschema/1-0-1`.
+
+<details>
+    <summary>Ecommerce action event properties</summary>
+
+| Request Key | Required | Type/Format | Description                                                                   |
+|-------------|----------|-------------|-------------------------------------------------------------------------------|
+| type        | Y        | string      | Specific ecommerce event type (automatically tracked).                                                |
+| name        | N        | string      | For `ProductListView` and `ProductListClick` events: human-readable list name |
+</details>
+
+The events are distinguished by their `type` property, which is different for each `Event` class tracked. Aside from the optional list `name` in the `ProductListView` and `ProductListClick` events, all tracked ecommerce properties are tracked as entities - see [below](#entities) for details.
+
+Check out the API docs ([Android](https://snowplow.github.io/snowplow-android-tracker/)) for the full details and method signatures.
+
+### ProductView
+
+Tracking a visit to a details screen for a specific product.
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="ios" label="iOS" default>
+
+Coming soon!
+
+  </TabItem>
+  <TabItem value="android" label="Android (Kotlin)">
+
+```kotlin
+val product = Product("productId", "category", "GBP", 100)
+val event = ProductView(product)
+
+tracker.track(event)
+```
+
+  </TabItem>
+  <TabItem value="android-java" label="Android (Java)">
+
+```java
+Product product = new Product("productId", "category", "GBP", 100);
+ProductView event = new ProductView(product);
+
+tracker.track(event);
+```
+
+  </TabItem>
+</Tabs>
+
+### AddToCart
+
+Tracking one or more products being added to a cart.
+
+<Tabs groupId="platform" queryString>
+  <TabItem value="ios" label="iOS" default>
+
+Coming soon!
+
+  </TabItem>
+  <TabItem value="android" label="Android (Kotlin)">
+
+```kotlin
+val product = Product("productId", "clothes/shirts", "EUR", 100.50)
+val event = AddToCart(listOf(product), 200, "EUR")
```

Comment:
I think the named parameters would be useful here too – it's not totally obvious to me what 200 means from the first glance. In Java, we could add comments to the parameters.